### PR TITLE
DEVTOOLS-49 Added detailed help messages for each command

### PR DESCRIPTION
**Pull request by fschargorosdky**
* Added a --help option to display a command help with full detail
* Changed some tests to resolve the container dependencies inside the test case (we have to change this in all tests)

### CODE REVIEW

File: lib/utils/messages.js [hunk 1]
```diff
@@ -30,15 +34,30 @@ module.exports = function () {
...
+    cleanupHelp: cleanupHelp,
     statusHelp: function () {
       return 'show the files status';
     },
     helpParameter: function () {
-      return 'for a detailed description of a command, add /?';
+      return 'for a detailed description of a command, use it with --help';
+    },
+    cleanupDetailedHelp: function () {
+      return cleanupHelp() + '\nAfter cleanup only create and command can be used' + requiresSetup;
```
Comment (juancoen) [inline lib/utils/messages.js]: 'create and setup'?

**Comment by juancoen** [comment]
please don't exceed 80 columns limit in messages.js

File: lib/utils/messages.js (hunk 1 above)
Comment (fschargorosdky) [inline lib/utils/messages.js]: done

**Comment by fschargorosdky** [comment]
Done with 
"please don't exceed 80 columns limit in messages.js"

File: lib/utils/messages.js [hunk 2]
```diff
@@ -4,17 +4,25 @@ var _ = require('lodash');
 var colors = require('colors/safe');
 
 var TOOL_NAME = 'api-sync';
+var requiresSetup = '\n\nRequires setup: Do it with ' + TOOL_NAME +
```
Comment (juancoen) [inline lib/utils/messages.js]: I think requiresSetup should only contain the message. The line end characters should be optionally included in the places where requiresSetup is used.

**Comment by juancoen** [comment]
looks good, just one thing

File: lib/utils/messages.js (hunk 2 above)
Comment (fschargorosdky) [inline lib/utils/messages.js]: Done

**Comment by juancoen** [comment]
looks good to me

**Pull request merged**